File: Linguagens de Programação/C/Fahrenheit2Celsius.c

#include <stdio.h>

int main()
{
    int n;
    printf("\tConversor de Fahrenheit para Celsius\n");
    printf("Insira a temperatura em Fahrenheit:");
    scanf("%d", &n);
    float celsius = ((n - 32) * 5) / (float)9;
    
    printf("A temperatura %d em celsius é: %.1f", n, celsius);
    
    return 0;
}

File: Linguagens de Programação/GOLang/fahrenheit2Celsius.go

package main
import ("fmt")

func main() {
    fmt.Println("Conversos de Fahrenheit para Celsius")
	fmt.Println("Digite a temperatura em graus Fahrenheit: ")
    var n int 
    fmt.Scanln(&n)
	var Celsius int
	
	Celsius = (n - 32) * 5 / 9
	fmt.Println("A temperatura em Celsius é: ", Celsius)
}
